apps/web/test/utils/bookingScenario/expects.ts

```diff
@@ -182,11 +182,15 @@ export function expectSuccessfulBookingCreationEmails({
   emails,
   organizer,
   booker,
+  guests,
```

Comment:
guests seems to be unused?

Reply:
Started using it. Plan was to add the verification for guest emails as well. Have added the support in the function, though guests isn't being passed as of now.